.github/workflows/tests.yml

```diff
@@ -85,10 +85,10 @@ jobs:
       - name: Adjust coverage file for Codecov
         run: bash <(sed -i 's/filename=\"/filename=\"pennylane\//g' coverage.xml)
 
-      - name: Upload coverage to Codecov
-        uses: codecov/codecov-action@v3
+      - uses: actions/upload-artifact@v2
         with:
-          file: ./coverage.xml
+          name: core-interfaces-coverage
+          path: ./coverage.xml

```

Comment:
Do we still want to upload here, or was this more so an attempt to get things to work?